### DISCUSSION TITLE
Adding Subprocess in a lane

### DISCUSSION THREAD

**Discussion by zarnejo-exers** [imixs/open-bpmn#331] [Q&A]
Good day!
I'm just wondering if it's possible to automatically create a subprocess and associate it with a lane. I looked around in the palette (via: https://open-bpmn.imixs.org/#/usr/src/app/open-bpmn.glsp-client/workspace), and I was not able to see anything that would allow me to add a subprocess (or a subtask). In this regard, is there an available call from the metamodel that would allow me to create a subprocess? 
Thank you in advance!

**Comment by rsoika** [comment]
Hi @zarnejo-exers , currently we do not yet support subtasks. Neither in the UI nor in the Meta model. 
We will work on this and I created a new feature-issue : https://github.com/imixs/open-bpmn/issues/332

**Reply by zarnejo-exers** [reply]
Ohh... This is noted. 
Thanks !
